Origin: vendor
Subject: enable crypto-nettle feature
Forwarded: not-needed

The sequoia-openpgp crate needs to have one of the crypto backends selected,
so hardcode this to nettle as Debian uses the nettle backend everywhere else.

diff --git a/Cargo.toml b/Cargo.toml
index 95d874b..b29d38c 100644
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -52,6 +52,7 @@
 [dependencies.sequoia-openpgp]
 version = "1.17"
 default-features = false
+features = [ "crypto-nettle" ]
 
 [dependencies.tempfile]
 version = "3"
